packages/leavitt-elements/src/leavitt-date-range.ts

```diff
@@ -57,24 +60,16 @@ export class LeavittDateRangeElement extends LitElement {
     }
   }
 
-  private _determineRange = (startDate, endDate) => {
-    let range = '';
-    Object.keys(dates).some(key => {
-      if (dates[key].startDate.format('YYYY-MM-DD') === startDate && dates[key].endDate.format('YYYY-MM-DD') === endDate) {
-        range = key;
-        return true;
-      }
-      return false;
-    });
-    return range;
-  };
+  async layout() {
+    return Promise.all([this.startDateField.layout(), this.endDateField.layout()]);
```

Comment:
Seems to be a issue downstream in mwc-textfield, but even calling layout on a previously hidden element only temporarily fixes the floating label.  As soon as its focused, the calculated text length is incorrect. 

![image](https://user-images.githubusercontent.com/9825025/149552805-a742678f-3efa-4cce-883f-2f4939b824a3.png)